content/blog/building-images-bake/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Creating a custom container image for CloudNativePG v2.0"
+date: 2025-06-17
+draft: false
+image:
+    url: 
+    attribution: 
+author: jgonzalez
+tags:
+ - blog
+ - information
+ - programming
+ - applications
+ - containers
+ - postgresql
+ - postgres
+ - images
+ - tutorial
+summary: Creating a container image for CloudNativePG Operator v2.0
+---
+
+## Summary
+In an almost [two years old blog post]({{% ref "/blog/creating-container-images/" %}}), we explained how
+to build a custom container image for CloudNativePG. After two years, many things have changed in the world of containers.
+One of those things has been the introduction of [Bake](https://docs.docker.com/build/) in Docker, which allows you to build
+images using a simple configuration file. Bake is now our recommended way to build images for CloudNativePG.
+
+We will follow a simple cooking recipe to create a custom container image or a set of container images, since Bake
```

Comment:
I think "cooking recipe" is a bit overkill in English. In Spanish, this is how you'd say it, but in English, recipe is enough.
Unless you were being playful, in which case you might consider "baking recipe".